ppr-ui/src/components/parties/party/SecuredParties.vue

```diff
@@ -108,249 +108,264 @@
-                      </v-chip>
-                    </div>
-                  </v-col>
-                </v-row>
-              </td>
-              <td>
-                <base-address
-                  :editing="false"
-                  :schema="addressSchema"
-                  :value="row.item.address"
-                />
-              </td>
-              <td>{{ row.item.emailAddress }}</td>
-              <td>{{ row.item.code }}</td>
-              <!-- Action Btns -->
-              <td class="actions-cell actions-width px-0">
-                <div
-                  class="actions float-right actions-up pr-4"
-                  v-if="
-                    isRegisteringParty(row.item) ||
-                      isSecuredPartyRestrictedList(registrationType) ||
-                      row.item.code > ''
-                  "
-                >
-                  <v-list
-                    class="actions__more-actions"
-                    :disabled="addEditInProgress"
-                  >
-                  <v-list-item
-                    v-if="(registrationFlowType === RegistrationFlowType.AMENDMENT
-                      && (row.item.action === ActionTypes.REMOVED || row.item.action === ActionTypes.EDITED) &&
-                      !isSecuredPartyRestrictedList(registrationType))
-                      || (isSecuredPartyRestrictedList(registrationType) && row.item.action === ActionTypes.ADDED
-                      && registrationFlowType === RegistrationFlowType.AMENDMENT)"
-                      class="v-remove"
-                      @click="undo(row.index)"
-                    >
-                      <v-list-item-subtitle>
-                        <v-icon small>mdi-undo</v-icon>
-                        <span class="ml-1 mr-2">Undo</span>
-                      </v-list-item-subtitle>
-                    </v-list-item>
-                    <v-list-item
-                      v-else-if="!isSecuredPartyRestrictedList(registrationType)
-                        || registrationFlowType === RegistrationFlowType.NEW"
-                      class="v-remove"
-                      @click="removeParty(row.index)"
+          <template v-slot:default>
+            <!-- Table Headers -->
+            <thead>
+              <tr>
+                <th v-for="header in headers" :key="header.value" :class="header.class">
+                  {{ header.text }}
+                </th>
+              </tr>
+            </thead>
+
+            <!-- Table Body -->
+            <tbody v-if="securedParties.length > 0">
+              <tr
+                v-for="(item, index) in securedParties"
+                :key="item.partyId"
+                class="party-row"
+                :class="{ 'disabled-text-not-action': item.action === ActionTypes.REMOVED}"
+              >
+                <!-- Edit Form -->
+                <template v-if="showEditParty[index]">
+                  <tr v-if="showEditParty[index]">
```

Comment:
Is the inner `v-if` necessary here?

Reply:
Good eye, probably not! Will have a look

Reply:
Fixed!